apis/v1alpha2/gateway_types.go

```diff
@@ -143,9 +146,7 @@ type GatewaySpec struct {
 }
 
 // Listener embodies the concept of a logical endpoint where a Gateway can
-// accept network connections. Each listener in a Gateway must have a unique
-// combination of Hostname, Port, and Protocol. This will be enforced by a
-// validating webhook.
+// accept network connections.
```

Comment:
// Listener embodies the concept of a logical endpoint where a Gateway ~can~ accepts network connections.